code/tika/TikaRestClient.php

```diff
@@ -28,19 +66,22 @@ public function isAvailable()
      */
     public function getVersion()
     {
-        $response = $this->get('version')->send();
-        // Parse output
-        if ($response->getStatusCode() == 200 &&
-            preg_match('/Apache Tika (?<version>[\.\d]+)/', $response->getBody(), $matches)
-        ) {
-            return (float)$matches['version'];
+        if (self::$_version == 0.0) {
+            $response = $this->get('version');
+            $response->setAuth(self::$_options['username'], self::$_options['password']);
+            $response->send();
+            // Parse output
+            if ($response->getResponse()->getStatusCode() == 200 &&
+                preg_match('/Apache Tika (?<version>[\.\d]+)/', $response->getResponse()->getBody(), $matches)
+            ) {
+                self::$_version = (float)$matches['version'];
+            } else {
+                self::$_version = 0.0;
+            }
         }
-
-        return 0.0;
+        return self::$_version;
```

Comment:
Yeah I'm starting to think that these types of things would be better as instance setter/getter methods